ext/Shared/Config.lua

```diff
@@ -1,22 +1,22 @@
 VEM_CONFIG = {
     -- OTHER --
-    PATCH_EXPLOSIONS_COLOR_CORRECTION = true, -- Disables color correction from explosions (explosions may cause a discontinue preset effect)
+    PATCH_EXPLOSIONS_COLOR_CORRECTION = false, -- Disables color correction from explosions (explosions may cause a discontinue preset effect)
 
     -- DAY-NIGHT --
     DN_SUN_TIMINGS = { 5.328, 21 },                                -- Sun times in hours (float) [sunrise, sunset]
     DN_PRESET_TIMINGS = { 0.222, 0.25, 0.29, 0.75, 0.833, 0.875 }, -- Always need to have the end time of the last preset in a day at the end
     DN_CHANGE_CLOUDS_SPEED_BASED_ON_DAY_LENGTH = false,            -- Synchronizes cloud speed to game time (useful for timelapses)
-    PATCH_DN_COMPONENTS = true,                                    -- Applies the needed Patches for Day-Night (removes bright meshes/textures meant for day)
+    PATCH_DN_COMPONENTS = false,                                   -- Applies the needed Patches for Day-Night (removes bright meshes/textures meant for day)
```

Comment:
?

Reply:
I've moved this ones to some settings in RM directly, apparently doing stuff with partitions in different mods can cause this kind of issues, so most of the logic where we were doing some partition data modifications were moved inside the RM repo in another file that manages this.

But I'd ike to keep the logic for this in VEM also, I for example usually run my local server without RM, so I just need to have this stuff and be able to control it with VEM setings.

Reply:
that´s new to me, we´ve been running this enabled for years now. You should only get problems if you try to edit the same partitions which shouldn´t the case for these.

Reply:
but go ahead as you like